projects/igniteui-angular/src/lib/select/select.component.ts

```diff
@@ -57,12 +75,28 @@ const noop = () => { };
         }
     `]
 })
-export class IgxSelectComponent extends IgxDropDownComponent implements IgxSelectBase, ControlValueAccessor, AfterContentInit,
-    EditorProvider {
+export class IgxSelectComponent extends IgxDropDownComponent implements IgxSelectBase, ControlValueAccessor,
+    AfterContentInit, OnInit, AfterViewInit, OnDestroy, EditorProvider {
 
     /** @hidden @internal do not use the drop-down container class */
     public cssClass = false;
 
+    /**
+     * @hidden @internal
+     */
+    @HostBinding('class.igx-input-group--valid')
```

Comment:
Same as the feedback in https://github.com/IgniteUI/igniteui-angular/pull/4916 - Not suitable to be using the group classes in the select itself.